web/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,14 @@
+export const environment = {
```

Comment:
https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public

Reply:
Même si c'est pas un enjeux de sécurité, la bonne pratique serait de ne pas rendre publique ces variables.

Reply:
Elles sont par définition publiques sur la page web, les projets web Firebase hosting ont aussi leur clé de visible
https://github.com/fireship-io/fireship.io/blob/master/components/src/environments/environment.prod.ts